apps/condo/domains/miniapp/schema/SendB2BAppPushMessageService.js

```diff
@@ -0,0 +1,207 @@
+const { GQLCustomSchema, itemsQuery, getByCondition, getById } = require('@open-condo/keystone/schema')
+
+const { NOT_FOUND } = require('@condo/domains/common/constants/errors')
+const access = require('@condo/domains/miniapp/access/SendB2BAppPushMessageService')
+const {
+    CONTEXT_FINISHED_STATUS,
+    DEFAULT_NOTIFICATION_WINDOW_DURATION_IN_SECONDS,
+    DEFAULT_NOTIFICATION_WINDOW_MAX_COUNT,
+    APP_BLACK_LIST_ERROR,
+} = require('@condo/domains/miniapp/constants')
+const { B2B_APP_MESSAGE_TYPES, MESSAGE_META } = require('@condo/domains/notification/constants/constants')
+const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
+const { RedisGuard } = require('@condo/domains/user/utils/serverSchema/guards')
+
+
+const redisGuard = new RedisGuard()
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+const ERRORS = {
+    APP_IN_BLACK_LIST: {
+        mutation: 'sendB2BAppPushMessage',
+        variable: ['data', 'app'],
+        code: FORBIDDEN,
+        type: APP_BLACK_LIST_ERROR,
+        message: 'The notification type is blocked for this app',
+    },
+    NO_B2B_CONTEXT: {
+        mutation: 'sendB2BAppPushMessage',
+        variable: ['data', 'app'],
+        code: FORBIDDEN,
+        type: NOT_FOUND,
+        message: 'No completed B2BContext in "Finished" status found for the provided organization and B2BApp.',
+    },
+    NO_B2B_APP_ACCESS_RIGHT: {
+        mutation: 'sendB2BAppPushMessage',
+        variable: ['data', 'app'],
+        code: FORBIDDEN,
+        type: NOT_FOUND,
+        message: 'No B2BAppAccessRight found for the provided user and B2BApp.',
+    },
+    USER_IS_NOT_AN_EMPLOYEE: {
+        mutation: 'sendB2BAppPushMessage',
+        variable: ['data', 'user'],
+        code: FORBIDDEN,
+        type: NOT_FOUND,
+        message: 'The provided user is not an employee of the specified organization.',
+    },
+    NO_B2B_APP_ROLE_FOR_EMPLOYEE_ROLE_AND_B2B_APP: {
+        mutation: 'sendB2BAppPushMessage',
+        code: FORBIDDEN,
+        type: NOT_FOUND,
+        message: 'No B2BAppRole found for the provided user, organization and B2BApp',
+    },
+}
+
+const SendB2BAppPushMessageService = new GQLCustomSchema('SendB2BAppPushMessageService', {
+    types: [
+        {
+            access: true,
+            type: `enum B2BAppMessageType { ${B2B_APP_MESSAGE_TYPES.join(' ')} }`,
+        },
+        {
+            access: true,
+            type: 'input SendB2BAppPushMessageInput { dv: Int!, sender: SenderFieldInput!, user: UserWhereUniqueInput!, organization: OrganizationWhereUniqueInput!, app: B2BAppWhereUniqueInput!, type: B2BAppMessageType!, meta: JSON!, }',
+        },
+        {
+            access: true,
+            type: 'type SendB2BAppPushMessageOutput { status: String!, id: String! }',
```

Comment:
Isn't status an enum?

Reply:
In the `sendMessage` it's a `String!`, i'm used same type here